.babelrc

```diff
@@ -35,6 +35,12 @@
       "plugins": [
         "./build/use-lodash-es"
       ]
+    },
+    "rollup": {
+      "plugins": [
+        "./build/use-lodash-es",
+        "external-helpers"
```

Comment:
No, I'm saying *don't* make them external. If you do, that becomes a dependency. `transform-runtime` inlines and centralizes helpers, so they don't need to be imported by the user, but are still efficient because there is only ever one copy of a helper.